code/game/machinery/iv_drip.dm

```diff
@@ -9,6 +9,19 @@
 	mouse_drag_pointer = MOUSE_ACTIVE_POINTER
 	var/obj/item/reagent_containers/iv_bag/bag = null
 
+/obj/machinery/iv_drip/Initialize(mapload)
+	. = ..()
+	if(mapload)
+		if(bag) // just in case anyone in the future ever adds IV drip subtypes with bags attached
+			return
+		for(var/obj/item/I in loc)
+			if(istype(I, /obj/item/reagent_containers/iv_bag))
+				I.forceMove(src)
+				bag = I
+				START_PROCESSING(SSmachines, src)
+				break
+	update_icon(UPDATE_OVERLAYS)
```

Comment:
Why is this one needed. Is it needed when a bag is linked? Or is it always needed? If it is only needed when a bag is linked then it would be best to add this method call to when you attach the found IV drip.